spec/features/users_show_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'James',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'Teacher from Mexico.'
+    )
+    @first_post = Post.create(author: @user, title: 'Hello, one', text: 'This is my first post')
+    @second_post = Post.create(author: @user, title: 'Hello, two', text: 'This is my second post')
+    @third_post = Post.create(author: @user, title: 'Hello, three', text: 'This is my third post')
+    @last_post = Post.create(author: @user, title: 'Hello, four', text: 'This is my last post')
+  end
+
+  describe 'user#index' do
+    before(:each) do
+      @second_user = User.create(
+        name: 'Ellen',
+        photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+        bio: 'Teacher from Poland.'
+      )
+      @first_post = Post.create(author: @second_user, title: 'Hello, there', text: 'This is my first post')
+      visit users_path
```

Comment:
- [ ] there should be specific/separate files for the tests of the user index, user show, post index and post show. please don't combine them in 1 file as you did now.
- [ ] please write the required tests for each file separately even though they overlap.